README.md

```diff
@@ -78,17 +74,17 @@ After you're finished please remove all the comments and instructions!
 
 - HTML
 - CSS
+- Javascript
 - Visual Studio Code 
-- Figma
+- Behance
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-- [Live Demo Link](https://joramot.github.io/capstone_one_global_summit/)
+- [Live Demo Link]( Comming soon!!!)
```

Comment:
- [ ] like the previous reviewer noted, deploying your project is a requirement. please make use of github pages to deploy the project and add a working link to the Readme file
- [ ] like the previous reviewer noted, please you are required to make a video presentation. present a video
   -  presenting the features of the project.
   - The maximum length of the video is 5 minutes.
   - The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
   - The video contains a demo of the project, showing the pages in different screen sizes.
   - The video contains a highlight about a piece of the code the student finds interesting or is proud of making.